sbi/inference/base.py

```diff
@@ -447,6 +450,17 @@ def _summarize(
             global_step=round_ + 1,
         )
 
+        # Add validation log prob for every epoch.
+        # Offset with all previous epochs.
+        epochs = torch.tensor(self._summary["epochs"][:-1])
+        offset = epochs.sum() if round_ > 0 else 0
```

Comment:
Not sure if you even need the `if` case.
```
a = torch.tensor([])
print(a.sum()) # -> tensor(0.)
```

Reply:
yes, good point, thanks!